cob_twist_controller/package.xml

```diff
@@ -30,10 +32,13 @@
   <run_depend>nav_msgs</run_depend>
   <run_depend>kdl_parser</run_depend>
   <run_depend>kdl_conversions</run_depend>
+  <run_depend>eigen_conversions</run_depend>
   <run_depend>orocos_kdl</run_depend>
   <run_depend>tf</run_depend>
   <run_depend>tf_conversions</run_depend>
   <run_depend>dynamic_reconfigure</run_depend>
+  <run_depend>cob_obstacle_distance</run_depend>
+  <run_depend>message_runtime</run_depend>
```

Comment:
```message_runtime``` is not needed as no messages/services are generated in cob_twist_controller package